### ISSUE TITLE
Codes in markdown is omitted

### ISSUE THREAD

**Issue by desperadoshi** [gera2ld/markmap-lib#3]
Markdown source file:

```
# Time Schemes

## Implicit Time Schemes

### BDF

### Comparative Study

- `YuMeilin-2019arXiv-CompareRosenbrock_ESDIRK_BDF`

## Multigrid Methods

```

Html output:
![nodecode](https://user-images.githubusercontent.com/9106886/79037931-ea8dd380-7c07-11ea-914f-7d1ca35db8e2.jpg)

`YuMeilin-2019arXiv-CompareRosenbrock_ESDIRK_BDF` is omitted in the html file.

markmap-lib version:

```
$ markmap --version
0.4.2
```

**Comment by shdq** [comment]
As a workaround, you can escape symbol with the backslash like this \`

**Comment by gera2ld** [comment]
fixed in 4301e9cf558855065c444740623c4cc1b308d286 , without style yet
